MapboxNavigationTests/Support/TokenTestViewController.swift

```diff
@@ -45,10 +45,11 @@ class TokenTestViewController: UIViewController {
         super.viewDidLoad()
         
         DispatchQueue.global().async {
-            self.directionsToken = Directions.skuToken
-            self.speechSynthesizerToken = SpeechSynthesizer.skuToken
             
             _ = self.semaphore.wait(timeout: .now() + 4)
+
+            self.directionsToken = Directions.skuToken
+            self.speechSynthesizerToken = SpeechSynthesizer.skuToken
```

Comment:
It results that polling these `skuToken`s in different moments of time fetches different token values...
That makes me think if that may result in single application be counted as 2 (or more?) users with different tokens? Could not come up with correct way to check that.